### PR TITLE
add 10.0.0

### PR DESCRIPTION
**Pull request by fennectech**
Done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 	<!--[if lt IE 9]>
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+		<script src="https://cdnjs.buttflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
```
Comment (HarryPeach) [inline index.html:18]: Looks good but you've (accidentally I hope) changed all instances of cloudflare to buttflare?

File: index.html [hunk 2]
```diff
@@ -220,7 +221,7 @@ <h4>Can I contribute?</h4>
 				<p class="paragraph">Yes! You can <a href="https://github.com/HarryPeach/bootscreennx">fork the repository</a> on GitHub and then submit a pull request.</p>
 		</div>
 
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+	<script src="https://cdnjs.buttflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```
Comment (HarryPeach) [inline index.html:224]: And here

File: index.html (hunk 1 above)
Comment (fennectech) [reply index.html:18]: CLOUD2BUTT HAHAHAHAHAHAHAHAHAHA

**Comment by fennectech** [comment]
Cloud2butt fun x3

**Comment by fennectech** [comment]
OMG   im litterly dying    I have cloud2butt on this computer!

**Pull request merged**